tests_v2/test_background_split_send_text_starts_terminal.py

```
@@ -0,0 +1,119 @@
+                "surface.send_text should start the background terminal without selecting its workspace",
+            )
+        finally:
+            try:
+                marker_path.unlink()
+            except FileNotFoundError:
+                pass
+            if created_workspace:
+                try:
+                    c.close_workspace(created_workspace)
+                except Exception:
+                    pass
```

Ruff (0.15.12)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)